doc/source/ray-observability/ray-logging.rst

```diff
@@ -4,6 +4,38 @@ Logging
 =======
 This document will explain Ray's logging system and its best practices.
 
+Internal Ray Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When ``import ray`` is executed, Ray's logging system is initialized, generating a sensible configuration given in ``python/ray/log.py``. The default logging level is ``logging.INFO``.
+
+Ray uses `rich <https://github.com/Textualize/rich>`__ to generate log messages that are well formatted and easily identified. If ``rich`` is not installed, a simple fallback formatter is used. Rich log messages display contextual information including the Ray component from which the message was generated and the filename and line number.
+
+Changing Ray's Internal Logging Configuration
```

Comment:
Can we just remove the whole section and just add how to configure library level logging through a  code example from the above section? I think everything here is unnecessary details for users. We need only these 3 information. 

1. All "ray" loggers are automatically configured and the config is specified in log.py
2. You can change the ray library logging config following the code below.
```
logger = logging.getLogger("ray")
logger # Modify the logging config
```
3. Add examples for other ray libraries (tune, data, rllib, train, and workflow). 
4. If you'd like to propagate the config to workers, follow `Worker logs` section